.changeset/great-baboons-lie.md

```diff
@@ -0,0 +1,7 @@
+---
+'@finos/legend-extension-dsl-data-space': patch
+'@finos/legend-query-builder': patch
+'@finos/legend-graph': patch
+---
+
+Add support for `MappingIncludeDataSpace`
```

Comment:
Probably the only one bump that requires this changeset message is with `'@finos/legend-extension-dsl-data-space': minor`